perf/perf_rawevents.py

```diff
@@ -63,7 +63,7 @@ def setUp(self):
             if not smm.check_installed(package) and not smm.install(package):
                 self.cancel('%s is needed for the test to be run' % package)
 
-        for filename in ['name_events_p8', 'raw_codes_p8', 'raw_codes_p9']:
+        for filename in ['name_events_p8', 'raw_codes_p8', 'raw_codes_p9', 'name_events_p10']:
```

Comment:
@disgoel  looks good to me with minor change like indentation above here (>79).
make sure you run pycodestyle before pushing PR so that you can catch them easily.